department-of-veterans-affairs/abd-vro
# Address some Secrel  issues

## What was the problem?


Associated tickets or Slack threads:
- #1579 

## How does this fix it?[^1]

* constrains version of com.fasterxml library to the patch
* Adds pip upgrade line to `Dockerfile-python`
* Bumps flyway version

Note that there is still some outstanding SecRel issues with these changes

## How to test this PR
- https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/6857162246 SecRel run


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
